src/contributing/void-docs/infradocs.md

```diff
@@ -0,0 +1,9 @@
+# InfraDocs
+
+[InfraDocs](https://infradocs.voidlinux.org/) is the manual for Void systems
```

Comment:
This sounds like infradocs is a manual for managing void systems in general.  This is in fact not true, is a meta-manual for managing void itself.